HandleNHNotificationCallActivity/__tests__/handler.test.ts

```diff
@@ -1,5 +1,5 @@
-/* tslint:disable: no-any */
-// tslint:disable-next-line: no-object-mutation
+/* eslint-disable @typescript-eslint/no-explicit-any */
+// eslint-disable-next-line 
```

Comment:
Avoid disable all rules, be specific on which rule you're going to disable. Other than being more communicative, it will allow prettier to work for rules you didn't mean to override.

Reply:
ok